### PR TITLE
Added some lines

### PR DESCRIPTION
**Pull request by 0Seppuku**

### CODE REVIEW

File: ITEM.tsv [hunk 1]
```diff
@@ -5422,70 +5422,70 @@ ITEM_20150714_005421	Research Results of Alwida
...
 ITEM_20150714_005481	Paz mushrooms
 ITEM_20150714_005482	Mushrooms which are often found in the Thorn Forest. It seems they must be cooked to be eaten.
-ITEM_20150714_005483	
+ITEM_20150714_005483	Ishibashi's rakteo
 ITEM_20150714_005484	The shape is very hard to be used as a weapon material.
-ITEM_20150714_005485	
-ITEM_20150714_005486	
-ITEM_20150714_005487	
-ITEM_20150714_005488	
+ITEM_20150714_005485	Poison thorn Chafperor
```
Comment (Sunaries) [inline ITEM.tsv:5485]: Should be: Poisonous Chafperor Thorn

File: ITEM.tsv [hunk 2]
```diff
@@ -5422,70 +5422,70 @@ ITEM_20150714_005421	Research Results of Alwida
...
+ITEM_20150714_005486	Chafperor visible on the forearm of the Meiji. The venom flows out.
```
Comment (Sunaries) [inline ITEM.tsv:5486]: Should be: A thorn found on the Chafperor's forearm. Venom oozes from it.

File: ITEM.tsv [hunk 3]
```diff
@@ -5422,70 +5422,70 @@ ITEM_20150714_005421	Research Results of Alwida
...
+ITEM_20150714_005487	Dark poison thorn tree stump
```
Comment (Sunaries) [inline ITEM.tsv:5487]: Should be: Concentrated Thorny Tree Stump Poison

File: ITEM.tsv [hunk 4]
```diff
@@ -5422,70 +5422,70 @@ ITEM_20150714_005421	Research Results of Alwida
...
+ITEM_20150714_005488	Thorns grow in the evil spirit. Surin has lodged a poisonous thorn.
```
Comment (Sunaries) [inline ITEM.tsv:5488]: Should be: A thorny tree that grows on wicked energies. Its thorns are poisonous.

File: ITEM.tsv [hunk 5]
```diff
@@ -5422,70 +5422,70 @@ ITEM_20150714_005421	Research Results of Alwida
...
 ITEM_20150714_005477	Complete Trap
-ITEM_20150714_005478	
+ITEM_20150714_005478	Evo trap of Nippon. The whole state is not yet operational.
 ITEM_20150714_005479	Scriptures Sheet
-ITEM_20150714_005480	
+ITEM_20150714_005480	Precious scriptures of the monks Wiley sheet. It seems scattered nanritong.
 ITEM_20150714_005481	Paz mushrooms
 ITEM_20150714_005482	Mushrooms which are often found in the Thorn Forest. It seems they must be cooked to be eaten.
-ITEM_20150714_005483	
+ITEM_20150714_005483	Ishibashi's rakteo
```
Comment (Sunaries) [inline ITEM.tsv:5483]: Should be: Rocktor's Stony Leg

File: ITEM.tsv [hunk 6]
```diff
@@ -5422,70 +5422,70 @@ ITEM_20150714_005421	Research Results of Alwida
...
 ITEM_20150714_005484	The shape is very hard to be used as a weapon material.
```
Comment (Sunaries) [inline ITEM.tsv:5484]: Should be: A piece of a Rocktor's leg. Can be used in weapons due to its shape and hardness.

File: ITEM.tsv [hunk 7]
```diff
@@ -5422,70 +5422,70 @@ ITEM_20150714_005421	Research Results of Alwida
...
 ITEM_20150714_005471	Explosion discovered in the Outpost area. It seems doeseo did not succeed.
 ITEM_20150714_005472	Dud Removal Tool Box
 ITEM_20150714_005473	Tools to remove the box contained a bomb
 ITEM_20150714_005474	Wedge needle
-ITEM_20150714_005475	
+ITEM_20150714_005475	Nancy Wedge pulled a sample of saliva from the bodies of the monks took the poison attack. If the hunter tracking colon may know the identity.
 ITEM_20150714_005476	Antidote that Hunters always have with them. It cures simple poisons immediately.
 ITEM_20150714_005477	Complete Trap
-ITEM_20150714_005478	
+ITEM_20150714_005478	Evo trap of Nippon. The whole state is not yet operational.
```
Comment (Sunaries) [inline ITEM.tsv:5478]: Should be: Evoniphon's trap. It's in perfect working condition.

File: ITEM.tsv [hunk 8]
```diff
@@ -5422,70 +5422,70 @@ ITEM_20150714_005421	Research Results of Alwida
...
 ITEM_20150714_005479	Scriptures Sheet
-ITEM_20150714_005480	
+ITEM_20150714_005480	Precious scriptures of the monks Wiley sheet. It seems scattered nanritong.
```
Comment (Sunaries) [inline ITEM.tsv:5480]: Should be: Monk Wiley's treasured sheet of scripture. Looks like they were scattered during the uproar.

File: ITEM.tsv [hunk 9]
```diff
@@ -5422,70 +5422,70 @@ ITEM_20150714_005421	Research Results of Alwida
...
 ITEM_20150714_005468	Explosive shot
-ITEM_20150714_005469	
+ITEM_20150714_005469	Explosive bombs that loud sounds great, but is reminded
 ITEM_20150714_005470	Dud
 ITEM_20150714_005471	Explosion discovered in the Outpost area. It seems doeseo did not succeed.
 ITEM_20150714_005472	Dud Removal Tool Box
 ITEM_20150714_005473	Tools to remove the box contained a bomb
 ITEM_20150714_005474	Wedge needle
-ITEM_20150714_005475	
+ITEM_20150714_005475	Nancy Wedge pulled a sample of saliva from the bodies of the monks took the poison attack. If the hunter tracking colon may know the identity.
```
Comment (Sunaries) [inline ITEM.tsv:5475]: Should be: A sample of the poisonous wedge needle used to attack the monks. The Hunter Tracking Master might be able to identify it.

**Comment by emailboxu** [comment]
https://github.com/Treeofsavior/EnglishTranslation/pull/501

Already made a lot of these changes near the bottom of the file. Just dumping the Korean into Google translate and copy pasta'ing it doesn't help the translation very much at all. If you can't get an accurate translation, please stick to editing other translations that need polishing instead.

**Comment by imcgames** [comment]
Thank you for your effort but it seems like a lot of it are done using machine translations and does not  the translate well so we will close this for now :smile:

**Pull request closed without merging**